applications/StructuralMechanicsApplication/python_scripts/adaptative_remeshing_structural_mechanics_analysis.py

```diff
@@ -120,6 +120,7 @@ def RunSolutionLoop(self):
                         self._GetSolver().Predict()
                         computing_model_part.Set(KM.MODIFIED, False)
                     computing_model_part.ProcessInfo.SetValue(KM.NL_ITERATION_NUMBER, non_linear_iteration)
+                    builder_and_solver.SetUpSystem(computing_model_part)
```

Comment:
i would say you need both SetupDofSet and SetupSystem. Alternatively you can simply call the Clear method of the strategy, and that will eb done automatically

Reply:
I will check that (if that is the case some implementations may be wrong, because I saw this alone in many places)

Reply:
Added